src/Sentry.AzureFunctions.Worker/Sentry.AzureFunctions.Worker.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+      <PackageTags>$(PackageTags);Azure;Functions;Worker</PackageTags>
+      <Description>Official Azure Functions Worker SDK integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
+      <Version>0.1.0-beta</Version>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Sentry.Extensions.Logging\Sentry.Extensions.Logging.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.13.0" />
+      <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
```

Comment:
Please set these to the _minimum_ (non-preview) version that our library requires.  (Unless any are marked as having vulnerabilities.)

Reply:
Probably this is `1.0.0` and `3.0.12`, but bump higher if those don't work.

Reply:
Can't use `Microsoft.Azure.Functions.Worker` `1.0.0` as it [didn't target `netstandard2.0`](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker/1.0.0#supportedframeworks-body-tab). The support for `netstandard2.0` was [added in `1.8.0`](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker/1.8.0#supportedframeworks-body-tab). But the `CancellationToken` support on `FunctionContext` was only added in [`1.9.0`](https://github.com/Azure/azure-functions-dotnet-worker/blame/1.9.0/src/DotNetWorker.Core/Context/FunctionContext.cs). Moving `Microsoft.Azure.Functions.Worker` to `1.9.0`.